specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2021-06-01-preview/examples/WaitStatisticsListByServer.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "api-version": "2021-06-01-preview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "testResourceGroupName",
+    "serverName": "testServerName",
+    "parameters": {
+      "properties": {
+        "observationStartTime": "2019-05-01T20:00:00.000Z",
+        "observationEndTime": "2019-05-07T20:00:00.000Z",
+        "aggregationWindow": "PT15M"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "636927606000000000-636927615000000000-send-wait/io/socket/sql/client_connection-2--0",
+            "name": "636927606000000000-636927615000000000-send-wait/io/socket/sql/client_connection-2--0",
```

Comment:
>636927606000000000-636927615000000000-send-wait/io/socket/sql/client_connection-2--0 [](http://example.com/codeflow?start=21&length=84)

A resource name cannnot contain '/'

Reply:
@ssen-msft I haven't made any changes to the existing definitions, do our resource names contain `/`?

Reply:
That's not a valid PG wait event; please delete it

Reply:
I modified the event name to remove the /s, we can always update the examples with more accurate names if needed without breaking anything